extensions/wikia/DesignSystem/bower_components/design-system/.bower.json

```diff
@@ -11,14 +11,13 @@
     "gulpfile.js"
   ],
   "homepage": "https://github.com/Wikia/design-system",
-  "version": "0.1.1",
-  "_release": "0.1.1",
+  "_release": "0dceecce3a",
   "_resolution": {
-    "type": "version",
-    "tag": "0.1.1",
-    "commit": "b9de8e29ade426f7db2bacbb32dbcda9731d4c0f"
+    "type": "branch",
+    "branch": "XW-1575",
+    "commit": "0dceecce3aec087d6cc34853e8487f55841f1ab8"
   },
   "_source": "git://github.com/Wikia/design-system.git",
-  "_target": "0.1.1",
+  "_target": "XW-1575",
```

Comment:
This should change after https://github.com/Wikia/app/pull/10999/files#r73507279 is addressed and new version installed.